### ISSUE TITLE
3.0 not working on WP 5.5 ("Array to string conversion")

### ISSUE THREAD

**Issue by mike-pt** [elementor/elementor#12231]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

Afeter updating to 3.0 I can't edit pages and get an error to see Elementor "system info".

## Steps to reproduce

Update to 3.0;
Edit a page;
Get and error, if WP_DEBUG is enabled it will show:
```
{"type":8,"message":
"Array to string conversion","file":........../wp-includes/formatting.php","line":1098}
```

Reverting to 2.99 works fine

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
== Server Environment ==
	Operating System: FreeBSD
	Software: nginx/1.18.0
	MySQL version: FreeBSD Ports v10.5.4
	PHP Version: 7.4.8
	PHP Max Input Vars: 3000
	PHP Max Post Size: 20M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5
	Site URL: XXXXXX
	Home URL: XXXXXX
	WP Multisite: No
	Max Upload Size: 20 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: pt-PT
	Timezone: Europe/Dublin
	Debug Mode: Active

== Theme ==
	Name: XXXXXX
	Version: 1.1
	Author: P-THEMES
	Child Theme: Yes
	Parent Theme Name: Porto
	Parent Theme Version: 5.4.4
	Parent Theme Author: P-THEMES
```
</details>

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.1

Feel free to update

Thanks!

**Comment by mike-pt** [comment]
Seems to be working, but I'm no longer seeing the option to edit Product Layouts 🤔  the theme also updated though so maybe that one is on their end
